uber/templates/regform.html

```diff
@@ -581,6 +581,15 @@ <h4 class="h4-inline">
     </div>
 </div>
 
+{% if attendee.is_unpaid %}
```

Comment:
I put a separate field here because `found_how` shouldn't be overloaded. I put it in regform.html on purpose -- we definitely want promo codes to work at-door, too.

Note that unlike most fields, this does not directly correspond with an attendee property, so we pass the variable `promo_code` explicitly to this template.